about.html

```diff
@@ -0,0 +1,206 @@
+          expand our invitation list including organizations and individuals.
+        </span>
+      </article>
+      <article id="desktop-about-text">
+        <span>
+          The Aruba Ruby Global Summit Global Summit brings together the
+          community of developers, academics, and CTO's who comprise the Ruby on
+          Rails network in a different country every two years.<br />This year,
+          we hope to expand our invitation list including organizations and
+          individuals who want to work with us on shared projects that advance
+          the cause of Ruby on Rails, software, technology culture and open
+          knowledge. So if you're active and engaged in the worlds of open
+          content and knowledge - free software advocates, wikipedians, Open
+          knowledge, galleries, libraries, museums, archives, governments and
+          foundations, lawyers, and activists - we hope you'll consider joining
+          us this year to build a stronger, more vibrant community together.
+        </span>
+      </article>
+      <article id="about-contact">
+        <p>
+          Please contact us per Email for any further questions about RoR Global
+          Summit 2023!
+        </p>
+        <p class="email">
+          <a href="mailto:hello@fakerails.org">hello@fakerails.org</a>
+        </p>
+      </article>
+    </section>
+
+    <section id="about-logo">
+      <h2>Aruba Ruby on Rails Global Summit 2023 Logo</h2>
+      <div id="about-logo-bar"></div>
+      <p id="about-logo-text">
+        The logo of Aruba Ruby on Rails Global Summit 2023 was decided through
+        the logo competition from 8, on December 23, 2022.
+      </p>
+      <p id="desktop-about-logo-text">
+        The logo of Aruba RoR Global Summit 2015 was decided through the logo
+        competition from 8, on December 23, 2022.. After online public voting,
+        the Selection Committee’s judging process, the logo from Naresh Agrawal
+        won the prize. After consultation with Mr. Agrawal, we optimized the
+        prize-winning-logo for this year’s summit as follows:
+      </p>
+
+      <img src="./assets/images/about_logo.png" />
+    </section>
+
+    <section id="about-previous">
+      <h2>See the past Aruba Ruby Global Summits</h2>
+      <div id="about-previous-bar"></div>
+      <p id="about-previous-text">
+        Take a look at the past two Aruba Ruby Global Summits
+      </p>
+      <div id="about-previous-images">
+        <article id="about-previous-2022">
+          <h3>2022</h3>
+          <h5>Aruba Ruby Conference</h5>
+        </article>
+        <article id="about-previous-2021">
+          <h3>2021</h3>
+          <h5>Aruba Ruby Conference</h5>
+        </article>
+      </div>
+    </section>
+
+    <section id="about-partners">
+      <h2>Partners</h2>
+      <div id="about-partners-bar"></div>
+      <div id="about-partners-logos">
+        <img src="./assets/images/mozilla.png" />
```

Comment:
Please add alt attributes for each `<img>` element in your pages as the previous reviewer requested. This is important for the accessibility and SEO of your web pages.